### PR TITLE
WIP: added statistics tab

### PR DESCRIPTION
**Pull request by ynixon**

### CODE REVIEW

File: app_s2i/templates/includes/_jumbotron.html [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <div class="container">
   <div style="background:transparent !important" class="jumbotron jumbotron-fluid">
-    <h1 class="display-4">Welcome to the DevOps Learning Community</h1>
+    <h1 class="display-5">Welcome to the DevOps Learning Community</h1>
```
Comment (lioramilbaum) [inline app_s2i/templates/includes/_jumbotron.html]: why this change was required. the jumbotron doesn't look good after it.

**Comment by lioramilbaum** [comment]
The tests should be updated as well. How do you know your feature works if you don't add tests to CI?

File: app_s2i/templates/includes/_jumbotron.html (hunk 1 above)
Comment (ynixon) [reply app_s2i/templates/includes/_jumbotron.html]: In my resolution the jumbotron takes about 50%
Annotation 2019-05-27 225837

**Comment by ynixon** [comment]
> The footer changed and now doesn't show the number but the logo of the statcounter service

The change of footer is done externally, via https://statcounter.com
I changed it back

File: app_s2i/templates/includes/_jumbotron.html (hunk 1 above)
Comment (lioramilbaum) [reply app_s2i/templates/includes/_jumbotron.html]: which is the purpose :-)

Comment (ynixon) [reply app_s2i/templates/includes/_jumbotron.html]: It is useful in the home tab. 
Why to have the same 50% heading on ALL tabs?

Comment (lioramilbaum) [reply app_s2i/templates/includes/_jumbotron.html]: Maybe you are right. I don’t know. Do we want to address it in this PR?

**Comment by lioramilbaum** [comment]
Can you add a test for the footer counter? To avoid such failures in the future.

**Comment by lioramilbaum** [comment]
Have you seen that there are merge conflicts? Please rebase and test in Dev and stage

**Comment by lioramilbaum** [comment]
@ynixon the PR doesn't pass CI. Can you please have a look?

**Comment by lioramilbaum** [comment]
@all-contributors please add @ShimiTaNaka for architecture

**Comment by lioramilbaum** [comment]
@all-contributors please add @ShimiTaNaka for infra

**Comment by ynixon** [comment]
This PR is still under work, how can change its status to WIP?
or should I withdraw it?

**Comment by ynixon** [comment]
When it will work for me, I will open a new PR
